packages/pds/src/services/moderation/views.ts

```diff
@@ -2,24 +2,36 @@ import { Selectable } from 'kysely'
 import { ipldBytesToRecord } from '@atproto/common'
 import { AtUri } from '@atproto/uri'
 import Database from '../../db'
+import { MessageQueue } from '../../event-stream/types'
 import { DidHandle } from '../../db/tables/did-handle'
 import { RepoRoot } from '../../db/tables/repo-root'
 import {
   View as RepoView,
   ViewDetail as RepoViewDetail,
 } from '../../lexicon/types/com/atproto/admin/repo'
-import { ViewDetail as RecordViewDetail } from '../../lexicon/types/com/atproto/admin/record'
-import { View as ActionView } from '../../lexicon/types/com/atproto/admin/moderationAction'
-import { View as ReportView } from '../../lexicon/types/com/atproto/admin/moderationReport'
+import {
+  View as RecordView,
+  ViewDetail as RecordViewDetail,
+} from '../../lexicon/types/com/atproto/admin/record'
+import {
+  View as ActionView,
+  ViewDetail as ActionViewDetail,
+} from '../../lexicon/types/com/atproto/admin/moderationAction'
+import {
+  View as ReportView,
+  ViewDetail as ReportViewDetail,
+} from '../../lexicon/types/com/atproto/admin/moderationReport'
 import { OutputSchema as ReportOutput } from '../../lexicon/types/com/atproto/report/create'
 import { ModerationAction, ModerationReport } from '../../db/tables/moderation'
 import { ActorService } from '../actor'
+import { RecordService } from '../record'
 
 export class ModerationViews {
-  constructor(private db: Database) {}
+  constructor(private db: Database, private messageQueue: MessageQueue) {}
 
   services = {
     actor: ActorService.creator(),
+    record: RecordService.creator(this.messageQueue),
```

Comment:
the messageQueue dependency makes a good argument for splitting out a RecordView service. but probably not necessary in this PR